the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
+
 		<!-- json-simple -->
-		
```

Comment:
NestedFramesPage:
 Should switch to frame based on the name of the frame rather than index.